README.md

```diff
@@ -1 +1,128 @@
+
+**Status**: active development
+**Source code**: https://github.com/BlueBrain/nexus-kg
+**Documentation**: https://bbp-nexus.epfl.ch/dev/docs/kg/
+
+### Nexus IAM (Identity and Access Managment)
+This service is a relay to any OAUTH2 identity service the platform maintainers decide to configure authentication against. The purpose of this service is to provide a unified identity service for the nexus platform irrespective of which actual underlying service is being used. Furthermore, this service is managing the privileges granted on entities to users.
+
+**Status**: active development
+**Source code**: https://github.com/BlueBrain/nexus-iam
+**Documentation**: coming soon
+
+### Nexus Commons
+This library is meant to contain all shared utilities of the nexus platform.
+Amongst other things, this library contains commons-types, sourcing-core, sourcing-akka, sourcing-mem, commons-service, commons-http, shacl-validator, sparql-client.
+
+**Status**: active development
+**Source code**: https://github.com/BlueBrain/nexus-commons
+**Documentation**: coming soon
+
+
+### sbt-nexus
+This library contains all the code related to the management of our Scala projects using [SBT](http://www.scala-sbt.org/). Its purpose is to promote reuse of build code across all projects.
+
+**Status**: active development
+**Source code**: https://github.com/BlueBrain/sbt-nexus
+**Documentation**: coming soon
+
+## Web applications
+
+### Nexus Search
+This web application allows users of the nexus platform to search in the knowledge graph. Beyond searching and inspecting data stored in the platform, its purpose is to enable the discovery of similar and related data.
+
+**Status**: active development
+**Source code**: https://github.com/BlueBrain/nexus-search-webapp
+**Documentation**: coming soon
+
+### Nexus Navigator
+This web component allows users of the platform to navigate graphically the knowledge graph data. A user can select an entity in the knowledge graph and display it neighboring entities in the form of a graph.
+
+**Status**: active development
+**Source code**: https://github.com/BlueBrain/nexus-navigator-webapp
+**Documentation**: coming soon
+
+### Nexus Registration
+This web application will allow users to enter data into the knowledge graph using forms generated automatically from domain’s schemas. This application is meant for low scale data integration while large scale will be done programmatically through APIs.
+
+**Status**: active development
+**Source code**: https://github.com/BlueBrain/nexus-registration-webapp
+**Documentation**: coming soon
+
+### Nexus WatchTower
+This web application will enable the close monitoring of an instance of Blue Brain Nexus and give vital information about its health and usage to its maintainers.
+
+**Status**: not started
+**Source code**: https://github.com/BlueBrain/nexus-registration-webapp
```

Comment:
Should this be linking to the registration-webapp?

Reply:
not at this stage I would say...